test/patterns/gen-ai/aws-text-to-sql/aws-text-to-sql.test.ts

```diff
@@ -0,0 +1,158 @@
+/**
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+import * as cdk from 'aws-cdk-lib';
+import { Template, Match } from 'aws-cdk-lib/assertions';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import { AwsSolutionsChecks } from 'cdk-nag';
+import {
+  TextToSql,
+  TextToSqlProps,
+  DatabaseType,
+  DbName,
+  MetatdataSource,
+} from '../../../../src/patterns/gen-ai/aws-text-to-sql';
+
+describe('Text to SQL Construct', () => {
```

Comment:
can we add another test to at least test the construction with the second DB available ?

Reply:
Moved DB out of the construct. d28bbe26726f810317c2b3533f33c248fd7c9cfb